extensions/VisualEditor/wikia/modules/ve/ui/widgets/ve.ui.WikiaAnonWarningWidget.js

```diff
@@ -0,0 +1,37 @@
+ve.ui.WikiaAnonWarningWidget = function VeUiWikiaAnonWarningWidget() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this function anonymous by removing its name: 'function() {...}'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)